mozci/push.py

```diff
@@ -299,6 +299,12 @@ def add(result):
         except MissingDataError:
             pass
 
+        # Gather task tags from the task table.
+        try:
+            add(run_query("push_tasks_tags_from_task", args))
```

Comment:
This will conflict with my data_sources PR, but I guess this is needed to enable manifest scheduling so should take priority.

Reply:
Sorry :'(
Hopefully it's reasonably easy to rebase your PR